### PR TITLE
Mish/button color and text change

### PR DESCRIPTION
**Pull request by mishs**
Changes to the Button and text color

### CODE REVIEW

**Comment by PiDelport** [comment]
@mishs: Which Trello card is this again?

**Comment by mishs** [comment]
> @mishs: Which Trello card is this again?
@PiDelport , here's the card:
 https://trello.com/c/ijPEI3bW/30-to-do-changes-to-homepage-section-from-matt

File: src/components/Hero/index.jsx [hunk 1]
```diff
@@ -86,6 +97,8 @@ const Hero = ({ classes, ...props }) => {
         <p>
           <Typography variant="headline" className={quote} component="span">
             “No one may be evicted from their home, or have their home demolished, without an order
+            of court<br />
```
Comment (PiDelport) [inline src/components/Hero/index.jsx]: Update from more recent discussion: this break should probably not be here.

**Comment by PiDelport** [comment]
Also: https://trello.com/c/7uP8DbYw/21-change-feedback-button-to-i-need-help-with-my-eviction

File: src/components/Hero/index.jsx [hunk 2]
```diff
@@ -86,6 +97,8 @@ const Hero = ({ classes, ...props }) => {
...
+            made after considering all relevant circumstances.”
```
Comment (PiDelport) [inline src/components/Hero/index.jsx]: We actually missed this error: The quotation text got duplicated.

**Pull request merged**